### PR TITLE
PLUGIN-1142: Display output schema only for Table Mode in ServiceNow Batch Source Plugin

### PR DESCRIPTION
**Pull request by sgarg-CS**
[PLUGIN-1142](https://cdap.atlassian.net/browse/PLUGIN-1142)

~**ServiceNow Multi-Source:** Display output schema only when data is requested for one table. No schema would be displayed if data is requested for more than one table~

**ServiceNow Batch Source:** Display output schema only for Table mode. No schema would be displayed for Reporting Mode

### CODE REVIEW

**Comment by sgarg-CS** [comment]
hi @rmstar 

Please review the changes in this PR.

File: src/main/java/io/cdap/plugin/servicenow/source/ServiceNowMultiSource.java [hunk 1]
```diff
@@ -75,7 +75,14 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
     // Since we have validated all the properties, throw an exception if there are any errors in the collector.
     // This is to avoid adding same validation errors again in getSchema method calls
     collector.getOrThrowException();
-    stageConfigurer.setOutputSchema(null);
```
Comment (rmstar) [inline src/main/java/io/cdap/plugin/servicenow/source/ServiceNowMultiSource.java]: We can keep the same behavior as other multi source plugins . ex - https://github.com/data-integrations/multi-table-plugins/blob/develop/src/main/java/io/cdap/plugin/MultiTableDBSource.java#L84
Revert the change in this file.

Comment (sgarg-CS) [reply src/main/java/io/cdap/plugin/servicenow/source/ServiceNowMultiSource.java]: Reverted 5b4f0c9cb9d3fb2d99126c231f4f9283f566e69a

**Review by rmstar — APPROVED** [review]
please squash commits

**Comment by sgarg-CS** [comment]
Commits squashed. Thanks!
